apps/site/assets/ts/stop/components/amenities/ElevatorsAmenityCard.tsx

```diff
@@ -1,13 +1,77 @@
 import React from "react";
-import AmenityCard from "./AmenityCard";
+import AmenityCard, { AmenityLink, AmenityModal } from "./AmenityCard";
 import { elevatorIcon } from "../../../helpers/icon";
+import { Alert, Facility } from "../../../__v3api";
+import Alerts from "../../../components/Alerts";
+import { hasFacilityAlert } from "../../../models/alert";
 
-const ElevatorsAmenityCard = (): JSX.Element => {
+const ElevatorsAmenityCard = ({
+  stopName,
+  alerts,
+  facilities
+}: {
+  stopName: string;
+  alerts: Alert[];
+  facilities: Facility[] | undefined;
```

Comment:
nit-pick(non-blocking): Could this just take in an empty array instead of undefined for no elevators?